nodejs/examples/02_create-tensor/index.js

```diff
@@ -0,0 +1,61 @@
+const Tensor = require('onnxruntime').Tensor;
+
+//
+// create a [2x3x4] float tensor
+//
+const buffer01 = new Float32Array(24);
+buffer01[0] = 0.1; // fill buffer data
+const tensor01 = new Tensor('float32', buffer01, [2, 3, 4]);
+// type 'float32' can be omitted and the type is inferred from data
+const tensor01_B = new Tensor(buffer01, [2, 3, 4]);
+
+//
+// create a [1x2] boolean tensor
+//
+const buffer02 = new Uint8Array(2);
+buffer02[0] = 1;  // true
+buffer02[1] = 0;  // false
+const tensor02 = new Tensor('bool', buffer02, [1, 2]); // type 'bool' cannot omit as both 'bool' and 'uint8' uses Uint8Array.
```

Comment:
If someone were to omit bool/uint8, then would they get an error as uint8 and bool share the same TypedArray ?

Reply:
yes there will be an Error thrown (runtime) / a compile error of no override found (compile time, if use typescript)